src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The stylesheet link should reference a CSS file, not an SCSS file. SCSS files are preprocessed into CSS, which is what browsers understand. Please change the href attribute to point to a compiled CSS file.